benchmarks/index.js

```diff
@@ -37,6 +37,16 @@ client.on('disconnect', (err) => {
   throw err
 })
 
+// http.get(httpOptions, (res) => {
+//   res
+//     .pipe(new Writable({
+//       write (chunk, encoding, callback) {
+//         console.log(`${chunk}`)
+//         callback()
+//       }
+//     }))
+//     .on('finish', () => console.log('done'))
+// })
```

Comment:
Why this is here?

Reply:
Sorry, I removed it. It was trash I forgot to remove.